middleware/etcd/setup.go

```diff
@@ -12,6 +10,7 @@ import (
 	"github.com/miekg/coredns/middleware"
 	"github.com/miekg/coredns/middleware/pkg/dnsutil"
 	"github.com/miekg/coredns/middleware/pkg/singleflight"
+	mwtls "github.com/miekg/coredns/middleware/tls"
```

Comment:
Oh, *should move*. I thought you were saying it *is already* under pkg. I will move it.